.gitlab-ci.yml

```diff
@@ -197,10 +197,10 @@ test-lin-dev-clang-cran:
     - >-
         Rscript -e 'l=tail(readLines("data.table.Rcheck/00check.log"), 1L); notes<-"Status: 2 NOTEs"; if (!identical(l, notes)) stop("Last line of ", shQuote("00check.log"), " is not ", shQuote(notes), " (size of tarball, installed package size) but ", shQuote(l)) else q("no")'
 
-## R 3.1.0
+## R 3.2.0
 # stated dependency on R
-test-lin-310-cran:
-  image: registry.gitlab.com/jangorecki/dockerfiles/r-3.1.0
+test-lin-320-cran:
```

Comment:
we could also make this `test-lin-ancient-cran` to make the diff smaller/less error-prone in the future. then the only change would need to be in the docker `image`.

Reply:
Yes, test name could be `test-lin-stated-cran`. Regarding naming docker image, I think it is still good to have 3.2.0 in the name of it, or eventually in the tag.

Reply:
Went with `minimum-r`. Agree it's good to keep `3.2.0` literal in the docker image name.